src/wallet/scriptpubkeyman.h

```diff
@@ -254,6 +254,13 @@ class ScriptPubKeyMan
     boost::signals2::signal<void ()> NotifyCanGetAddressesChanged;
 };
 
+/** OutputTypes supported by the LegacyScriptPubKeyMan */
+static const std::unordered_set<OutputType> LEGACY_OUTPUT_TYPES {
```

Comment:
Tried that but it doesn't work.

Reply:
You can't `constexpr` things that need allocation. You could use an array instead of an unordered_set, and use `std::find` to look up (which will likely be faster too, due to the tiny size), but it really doesn't matter here.